MiniApp/Classes/JavascriptBridge/MiniAppJavaScriptEnumerations.swift

```diff
@@ -55,3 +56,20 @@ public enum MiniAppCustomPermissionGrantedStatus: String, Codable {
         }
     }
 }
+
+enum MiniAppInterfaceOrientation: String, Codable {
+    case lockPortrait = "rakuten.miniapp.screen.LOCK_PORTRAIT"
+    case lockLandscape = "rakuten.miniapp.screen.LOCK_LANDSCAPE"
+    case lockRelease = "rakuten.miniapp.screen.LOCK_RELEASE"
+
+    public var orientation: UIInterfaceOrientationMask {
+        switch self {
+        case .lockPortrait:
+            return .portrait
+        case .lockLandscape:
+            return .landscape
+        case .lockRelease:
+            return .all
```

Comment:
Could this override the Host App's default orientation setting? For example, this scenario: 
1. The App's view controller is locked to portrait mode by default
2. Then the mini app locks to landscape mode
3. Then the mini app releases the lock
4. The SDK sets the view controller to `.all`

Would this cause the App's view controller to no longer be locked to portrait and enable auto-rotate? Is there a way to save the App's original orientation and set it when the lock is released?

Reply:
This won't override the host app default orientation, returning `.all` would reset the host app to its orientation once the lock is released. However, it is not good to return `.all` without knowing the host app current orientation. I have updated the code now to return the `.landscape` or `.portrait` based on the device rotation. Thank you 😄